pulsar-common/src/test/java/org/apache/pulsar/common/util/netty/DnsResolverTest.java

```diff
@@ -18,13 +18,42 @@
  */
 package org.apache.pulsar.common.util.netty;
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.testng.Assert.assertEquals;
 import io.netty.channel.EventLoop;
 import io.netty.resolver.dns.DnsNameResolverBuilder;
+import java.security.Security;
+import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class DnsResolverTest {
+    private static final int MIN_TTL = 0;
+    private static final int TTL = 101;
+    private static final int NEGATIVE_TTL = 121;
+
+    @BeforeClass
```

Comment:
```suggestion
    @BeforeClass(alwaysRun = true)
```
TestNG has some surprising behaviors unless `alwaysRun=true` is used. It happens when using test groups. Because of that, it's better to always add `alwaysRun = true to BeforeClass/AfterClass.